docker-compose.connect-stack.yml

```diff
@@ -0,0 +1,34 @@
+version: '2.3'
+
+services:
+  openvpn-client:
+    environment:
+      - DESECSTACK_VPN_SERVER=openvpn-server
+    networks:
+      vpnconnect:
+  openvpn-server:
+    networks:
+      vpnconnect:
+  replicator:
+    environment:
+      - DESECSTACK_VPN_SERVER=desec.$DESECSTACK_DOMAIN
+      - DESEC_NS_E2E2=test mode
```

Comment:
Does the string value mean anything? (Else, `1` would be a more canonical value.)

Reply:
done